frontend/src/components/StockNews/StockNews.jsx

```diff
@@ -1,105 +1,97 @@
-                value={currentStock?.ticker || ""}
-                onChange={handleOptions}
+            <DropdownButton
+                title={currentStock === null ? "Select a Stock" : currentStock}
+                className="mt-2 mb-3"
             >
-                <option value="" disabled defaultValue>
-                    Select a Stock You Follow
-                </option>
-                {stocks?.map((stock) => (
-                    <option key={stock.ticker} value={stock.ticker}>
-                        {stock.ticker}
-                    </option>
-                ))}
-            </select>
-            <div className="news">
-                {stockNews?.map((article) => (
-                    <div className="article-margin" key={article.url}>
-                        <a
-                            href={article.url}
-                            target="_blank"
-                            rel="noopener noreferrer"
-                            className="article-link"
+                {stocks?.map((stock) => {
+                    return (
+                        <Dropdown.Item
+                            key={stock.ticker}
+                            onClick={() => setCurrentStock(stock.ticker)}
                         >
-                            <div className="article">
-                                <div className="article-logo">
-                                    <img
-                                        src={currentStock.logo}
-                                        alt={`This is a logo of ${currentStock.name}.`}
-                                    />
-                                </div>
-                                <div className="article-content">
-                                    <h2>{article.title}</h2>
-                                    <p>{article.content}</p>
-                                    <p>
-                                        {formatDate(article.publishedAt)} •{" "}
-                                        {article.author}
-                                    </p>
-                                </div>
-                            </div>
-                        </a>
-                    </div>
-                ))}
-            </div>
+                            {stock.ticker}
+                        </Dropdown.Item>
+                    );
+                })}
+            </DropdownButton>
+            {stockNews?.slice(0, articlesToShow).map((article) => {
+                return (
+                    <Card
+                        key={article.id}
+                        className="mb-5 cursor-pointer"
+                        onClick={() => window.open(article.url, "_blank")}
+                        style={{ cursor: "pointer" }}
+                    >
+                        <Card.Img variant="top" src={article.image} />
+                        <Card.Body>
+                            <Card.Text>{article.headline}</Card.Text>
```

Comment:
nit: Can you use `Card.Title` instead of `Card.Text` to make the title of the article more prominent? Sometimes articles will do a preview of one-two sentences and that would make sense in the `Card.Text`